butter_cms/post.py

```diff
@@ -10,10 +10,15 @@ def __init__(self, auth_token):
     def all(self, params=None):
```

Comment:
Does this call result in a trailing slash? `v2/posts/?...`

Reply:
It should. The path for `posts/` is set on https://github.com/ButterCMS/buttercms-python/blob/master/butter_cms/post.py#L8